### DISCUSSION TITLE
DKIM record check error

### DISCUSSION THREAD

**Discussion by fernandoSynetic** [postalserver/postal#2999] [Help with using Postal] [answered]
Hello. I try to validate a DKIM record but get an error. Validating with online tools - https://www.mail-tester.com/spf-dkim-check - everything seems to be fine. Can you help me?
Thanks!
---

DKIM Record

WARNING The DKIM record at postal-ZmoVpd._domainkey.metawayenergy.com does not match the record we have provided. Please check it has been copied correctly.

You need to add a new TXT record with the name postal-ZmoVpd._domainkey with the following content.

v=DKIM1; t=s; h=sha256; p=MIGfMA0GCSqGSIb3DQEBAQUAA4GNADCBiQKBgQDZzm88UBQ+GbEnv6uEaWr3BLQ70K5Jl7969Ph4zMYWpHqGqlu7uqxy4GDYNKTJFxQsJ0XUicoPANa2DR22NNSUpEZsrZqMfiXFw5E7KU2bTWgzJ 6G6biwhnXePpvDMYt1NxbN59hBjarQLIcGpk0VpLvCm3jlpcAdYHd5Kkx10yQIDAQAB;

**Comment by willpower232** [comment, marked as answer]
It may be that you have to toggle the `local ns` setting in the config file. You could also verify by SSHing to your server and running `dig txt postal-ZmoVpd._domainkey.metawayenergy.com`
